assets/js/utils/get-query.js

```diff
@@ -76,5 +76,8 @@ export default function getQuery( blockAttributes, name ) {
 			break;
 	}
 
+	// Always only show visible products.
+	query.catalog_visibility = 'visible';
```

Comment:
If we do go with the below additional `hidden` option I mentioned, we may want to add in an additional `catalogVisiblity` attribute and only set `query.catalog_visibility` if nothing is passed.

Reply:
We probably won't need to control that from the block, so I don't think this is necessary.

Reply:
Can you move this up to the initial settings around line 15? That's where we set up all the default values.